tests/client/test_client.py

```diff
@@ -252,12 +252,74 @@ def test_flag_columns(self):
 
         flags = [True, False, False]
         client = Client(URL_ROOT)
-        summary = client.flag_columns(flags)
+        summary = client.flag_columns(flags, "inferenceable")
 
         self.assertEqual({"inferenceable": flags}, summary.column_flags)
 
-    def test_flag_columns_not_inferenceable(self):
-        """Flags not explicitly titled "inferenceable" are not allowed."""
+    @responses.activate
```

Comment:
I don't really like that I copied the entire test and modified it for the enrichable case. Feels like we could consolidate the two, but I didn't have time to do more on that.

Reply:
for tests, I'm generally fine with not getting too DRY, if it couples things, or makes it prone to erroring for > 1 reason.

Reply:
Just here to say "I agree with @tuddman" DRY is for source code, but if I have to push the mental stack to understand what a test is doing, I die a little inside.